Bottom: 164b7958644b5dfc1c4932984370c55c087f9d6c
Top:    b29cd3859d0e0b93b6ce2ff0cf42381921f320ac
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 01:17:24 +0100

Avoid clearing pins / vias from polygons... it is already done!




---

diff --git a/src/draw.c b/src/draw.c
index daa4a38..37c1dfe 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1178,6 +1178,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1200,6 +1201,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
